numpy/__init__.py

```diff
@@ -120,13 +120,90 @@
     __all__ = ['exceptions']
 
     from . import core
-    from .core import *
+    from .core import (
+        _no_nep50_warning, memmap,
+        iinfo, finfo,  # getlimits submodule exports
```

Comment:
no need for this comment I think?

Reply:
Sure, I removed it! I think the list of imports will change - I will perform formatting and sorting in the end.